glide.yaml

```diff
@@ -44,3 +44,8 @@ import:
   - quantile
 - package: github.com/uber-go/atomic
 - package: github.com/prashantv/protectmem
+- package: github.com/opentracing/opentracing-go
+- package: github.com/uber/jaeger-client-go
+- package: github.com/opentracing/basictracer-go
+- package: github.com/gogo/protobuf
```

Comment:
a fully functional mock tracer is not available directly in opentracing-go, so I replaced basictracer with it altogether.

btw, the protobuf dependency had to be in glide.yaml as otherwise it was not getting installed or even locked in glide.lock, even though basictracer depends on it. Not an issue now.